From 926a1d9c8eb44e55358a4e0dd1586a7fbdfe02ab Mon Sep 17 00:00:00 2001
From: Anders Berg <anders.berg@lsi.com>
Date: Tue, 4 Jun 2013 15:17:34 +0200
Subject: [PATCH 67/75] arm/mach-axxia: Corrected definition of
 __arch_iounmap.

Fixed unresolved reference to __iounmap when building modules. Macro
__arch_iounmap must point to an exported symbol.

Signed-off-by: Anders Berg <anders.berg@lsi.com>
---
 arch/arm/mach-axxia/include/mach/io.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-axxia/include/mach/io.h b/arch/arm/mach-axxia/include/mach/io.h
index 40924cd..a4eddd9 100644
--- a/arch/arm/mach-axxia/include/mach/io.h
+++ b/arch/arm/mach-axxia/include/mach/io.h
@@ -34,6 +34,6 @@
 
 void __iomem *__axxia_arch_ioremap(phys_addr_t, size_t, unsigned int);
 #define __arch_ioremap __axxia_arch_ioremap
-#define __arch_iounmap __iounmap
+#define __arch_iounmap __arm_iounmap
 
 #endif
-- 
1.7.5.4

